pkg/backend/llm/gpt4all/gpt4all.go

```diff
@@ -60,3 +60,12 @@ func (llm *LLM) PredictStream(opts *pb.PredictOptions, results chan string) erro
 
 	return nil
 }
+
+func (llm *LLM) Unload() error {
```

Comment:
Let's not add unused code - I know it's tempting, but then it makes it not obvious what the code actually do for first time readers - is it used? where?